src/main/java/interface_adapter/list/ListController.java

```diff
@@ -23,4 +28,12 @@ public void executeCollectionSwitch(String desiredCollectionName) {
         listInputData.setNameOfDesiredCollection(desiredCollectionName);
         listInteractor.execute(listInputData);
     }
+
+    /**
+     * Filters current list based on recommendations.
+     *
+     * @param filter the key we want to filter for
+     */
+    public void filterMoviesByDescription(String filter) {
+    }
```

Comment:
I made my own controller for this purpose, so you can discard this method since it isn't used (unless it's here for some interface or testing purpose).

```suggestion

```